src/langchain_google_spanner/prompts.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GQL_EXAMPLES = """
+The following query in backtick matches all persons in the graph FinGraph
+whose birthday is before 1990-01-10 and
+returns their name and birthday.
+```
+GRAPH FinGraph
+MATCH (p:Person WHERE p.birthday < '1990-01-10')
+RETURN p.name as name, p.birthday as birthday;
+```
+
+The following query in backtick finds the owner of the account with the most
+incoming transfers by chaining multiple graph linear statements together.
+```
+GRAPH FinGraph
+MATCH (:Account)-[:Transfers]->(account:Account)
+RETURN account, COUNT(*) AS num_incoming_transfers
+GROUP BY account
+ORDER BY num_incoming_transfers DESC
+LIMIT 1
+
+NEXT
+
+MATCH (account:Account)<-[:Owns]-(owner:Person)
+RETURN account.id AS account_id, owner.name AS owner_name, num_incoming_transfers;
+```
+
+The following query finds all the destination accounts one to three transfers
+away from a source Account with id equal to 7.
+```
+GRAPH FinGraph
+MATCH (src:Account {{id: 7}})-[e:Transfers]->{{1, 3}}(dst:Account)
+RETURN src.id AS src_account_id, dst.id AS dst_account_id;
+```
+Carefully note the syntax in the example above for path quantification,
+that it is `[e:Transfers]->{{1, 3}}` and NOT `[e:Transfers*1..3]->`
+"""
+
+DEFAULT_GQL_TEMPLATE_PART0 = """
+Create an ISO GQL query for the question using the schema.
```

Comment:
I wonder whether we should just say `Spanner Graph GQL` because I assume there will be many vendors of ISO GQL that are slightly different here and there. Maybe be explicit about Spanner here.

Reply:
done